cloudbuild.yaml

```diff
@@ -0,0 +1,58 @@
+steps:
+  - id: 'pull_latest_image'
+    name: 'gcr.io/cloud-builders/docker'
+    args: ['pull', 'gcr.io/$PROJECT_ID/gobjob-$_DEPLOY_ENV:latest']
+
+  - id: 'build'
+    name: 'gcr.io/cloud-builders/docker'
+    args: [
+        'build',
+        '-t', 'gcr.io/$PROJECT_ID/gobjob-$_DEPLOY_ENV:$SHORT_SHA',
+        '-t', 'gcr.io/$PROJECT_ID/gobjob-$_DEPLOY_ENV:latest',
+        '--cache-from', 'gcr.io/$PROJECT_ID/gobjob-$_DEPLOY_ENV:latest',
+        '.',
+    ]
+  - id: 'push'
+    name: 'gcr.io/cloud-builders/docker'
+    args: [
+        'push',
+        'gcr.io/$PROJECT_ID/gobjob-$_DEPLOY_ENV:$SHORT_SHA'
+    ]
+
+  - id: 'push2'
```

Comment:
I tried literally everything to get cloud build to correctly tag the image with both its $SHORT_SHA hash AND `latest`. The only way I got it to work is just to push it twice so whatever